tests/shell/shell_recorder/tutorial_wrapper/README.md

```diff
@@ -20,7 +20,7 @@ kdb rm /tests/markdown/babymetal
 ```
 
 . The test above invokes three commands. The first command stores the value `death` in the key with the name
-`user/tests/markdown/napalm`. The special comment `#> ` below the command specifies the expected output to the standard
+`user/tests/markdown/napalm`. The special comment `#>` below the command specifies the expected output to the standard
```

Comment:
This is a semantic change (different string), is this on purpose?

Reply:
I also noticed that update. I did not think that it was that big of a problem. Anyway, I adapted the text in commit b358db7. Now it does not matter that Prettier removes the trailing whitespace from the code span.